spec/features/dashboard_spec.rb

```diff
@@ -217,6 +219,28 @@
         expect(page).to_not have_content(other_solicitor_dr.solicitor_name)
       end
     end
+
+    scenario 'after the refresh i can still see ONLY the correct data', js: true do
+      visit defence_requests_path
+
+      within ".accepted_defence_request" do
+        expect(page).to have_content(solicitor_dr.detainee_name)
+        expect(page).to_not have_content(solicitor_dr_not_accepted.detainee_name)
+      end
+      expect(page).to_not have_selector(".created_defence_request")
```

Comment:
Really don't like setting expectations on styles like this - makes tests very brittle to design changes, but I guess it's okay for the alpha